### ISSUE TITLE
latest libmpv build crashes

### ISSUE THREAD

**Issue by stax76** [shinchiro/mpv-winbuild-cmake#112]
The latest x64 libmpv build mpv-dev-x86_64-20210316-git-5824d9f crashes and dies silently on startup. There is an error entry in the Windows 10 application event log:

Fault bucket 1160556840952446317, type 4
Event Name: APPCRASH
Response: Not available
Cab Id: 0

Problem signature:
P1: mpvnet.exe
P2: 5.4.8.7
P3: 6065a985
P4: ntdll.dll
P5: 10.0.19041.844
P6: 60a6ca36
P7: c0000028
P8: 0000000000102136

The player build mpv-x86_64-20210316-git-5824d9f works fine and the previous libmpv build mpv-dev-x86_64-20210228-git-d1be8bb works fine too.

mpv_initialize is the function where it crashes and dies, here is the error from Visual Studio:

Unhandled exception at 0x00007FFD6C712136 (ntdll.dll) in mpvnet.exe: 0xC0000028: An invalid or unaligned stack was encountered during an unwind operation.

I hope the next build is fine, if it's not then I hope it's not hard to find the cause.

I tried to start with defaults not using my mpv.conf, but it did not help.

**Comment by shinchiro** [comment]
Previous build used binutils 2.36.x which might introduced the problem. Try again with today build

**Comment by stax76** [comment]
Thanks for the new build, I've tried it, it crashes too. Is there anything else I can try?

**Comment by avih** [comment]
Could it be related to js?
https://github.com/mpv-player/mpv/issues/8667

TL;DR: mpv mostly exits silently when trying to load js scripts, not fixed in latest @shinchiro build either. Can't reproduce with my own builds.

**Comment by stax76** [comment]
I confirm my issue is indeed JS related.
